gulpfile.js

```diff
@@ -67,13 +67,13 @@ function transpileBowerComponents(config) {
     '!www/bower_components/webcomponentsjs/**/*.js',
     '!www/bower_components/webcomponentsjs/**/*.html', '!www/bower_components/**/*.min.js'
   ];
-  const bowerComponentsDest = `${config.targetDir}/bower_components`;
```

Comment:
Almost! There was still one use, for `--watch`. Since that complicates the Gulpfile significantly and is, I think, much less used now that we can quickly test in Electron, I killed both in one go. WDYT?